nixos/modules/services/backup/znapzend.nix

```
@@ -315,6 +315,14 @@ in
         '';
       };
 
+      mailErrorSummaryTo = mkOption {
+        type = str;
```

```suggestion
        type = singleLineStr;
```

while you're at it ;)